java/arcs/core/policy/PolicyTranslation.kt

```diff
@@ -0,0 +1,37 @@
+package arcs.core.policy
+
+import arcs.core.analysis.RecipeGraph
+
+/**
+ * Translates the given [Policy] into dataflow analysis checks and claims on the given
+ * [RecipeGraph], returning a modified copy of the graph.
+ */
```

Comment:
Don't we want to refer to the argument names instead of types here?
`[Policy]` -> `[policy]`
`[RecipeGraph]` -> `[graph]`

Reply:
Done